Steamworks/src/org/usfirst/frc/team4786/robot/subsystems/DriveTrain.java

```diff
@@ -181,6 +206,37 @@ public void driveWithVelocityControl(double targetSpeedLeft, double targetSpeedR
 
 	}
 	
+	//Methods called by TurnToAngle
+	public void turnToAngleInit(double targetAngle){
+		/* PIDController calculates a rate of motor output,
+		 * so the CANTalons need to be in PercentVbus mode */
+		frontLeft.changeControlMode(TalonControlMode.PercentVbus);
+		frontRight.changeControlMode(TalonControlMode.PercentVbus);
+		
+		//Initialize turnController and set the target
+		navX.reset();
```

Comment:
You should probably comment somewhere that `TurnToAngle` will always turn relative to the angle the robot is facing when the command is started.

If you want the turning to be to an angle relative to the field (like 30 degrees from the alliance wall), you need to move this `navX.reset()` call to the constructor of the  `DriveTrain` class.